docs/docs.scss

```diff
@@ -19,6 +19,12 @@ $snippet-border: 2px solid #CCC;
 $icon-block-bg: #eee;
 $icon-block-size: 75px;
 
+.daptiv-grid{
+    .box-column {
```

Comment:
it feels nice as a modifier. then you can have `outlined column` instead of `box-column column`